docs/sources/alerting/set-up/provision-alerting-resources/export-alerting-resources/index.md

```diff
@@ -58,27 +68,89 @@ To export a modified alert rule without saving the modifications, complete the f
+
+### Export contact points
+
+To export contact points from the Grafana UI, complete the following steps.
+
+1. Click **Alerts & IRM** -> **Contact points**.
+1. Find the contact point you want to export and click **More** -> **Export**.
+1. Choose the format to export in.
+
+   The exported contact point appears in different formats - YAML, JSON, Terraform.
 
 1. Click **Copy Code** or **Download**.
 
-   a. Choose **Copy Code** to go to an existing file and paste in the code.
+### Export templates
+
+Grafana currently doesn't offer an Export UI for notification templates, unlike other Alerting resources presented in this documentation.
 
-   b. Choose **Download** to download a file with the exported data.
+However, you can export it by manually copying the content template and title directly from the Grafana UI.
 
-## Export API endpoints
+1. Click **Alerts & IRM** -> **Contact points** -> **Notification templates** tab.
+1. Find the template you want to export.
+1. Copy the content and title.
+1. Adjust it for the [file provisioning format][alerting_file_provisioning_template] or [Terraform resource][alerting_tf_provisioning_template].
 
-You can also use the **Alerting provisioning HTTP API** to export alerting resources in YAML or JSON formats for provisioning.
+### Export the notification policy tree
 
-Note that most Alerting endpoints return a JSON format that is not compatible for provisioning via configuration files, except the ones listed below.
+All notification policies are provisioned through a single resource: the root of the notification policy tree.
 
-| Method | URI                                                              | Summary                                                                                  |
-| ------ | ---------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
-| GET    | /api/v1/provisioning/alert-rules/:uid/export                     | [Export an alert rule in provisioning file format.][export_rule]                         |
-| GET    | /api/v1/provisioning/folder/:folderUid/rule-groups/:group/export | [Export an alert rule group in provisioning file format.][export_rule_group]             |
-| GET    | /api/v1/provisioning/alert-rules/export                          | [Export all alert rules in provisioning file format.][export_rules]                      |
-| GET    | /api/v1/provisioning/contact-points/export                       | [Export all contact points in provisioning file format.][export_contacts]                |
-| GET    | /api/v1/provisioning/policies/export                             | [Export the notification policy tree in provisioning file format.][export_notifications] |
+{{% admonition type="warning" %}}
+
+Since the policy tree is a single resource, provisioning it will overwrite a policy tree created through any other means.
+
+{{< /admonition >}}
```

Comment:
Here is mismatched syntax (`%` to open `<` to close) but apparently it is fine, I checked locally.